node_modules/col-assets/lib/api.js

```diff
@@ -1265,7 +1265,7 @@ var createComment = module.exports.createComment = function(ctx, assetId, body,
           }
 
           // Retrieve the created comment, including the associated user
-          return getComment(comment.id, function(err, comment) {
+          getComment(comment.id, function(err, comment) {
```

Comment:
@johncrossman - Yes, this `return` was superfluous as the real `return` happens one level deeper inside the function.